tensorflow/contrib/layers/python/layers/layers_test.py

```diff
@@ -3688,7 +3688,17 @@ def test_1d_invalid_input(self):
                                    'rank of x must be at least 2 not: 1'):
         x = constant_op.constant([[]], shape=[0])
         _layers.legacy_fully_connected(x, 2, activation_fn=nn_ops.softmax)
-
-
+  
+class zero_debias_moving_mean(test.TestCase):
```

Comment:
In this case, you can just add the test to the existing BatchNormTest suite, no need to add a new ones.

In general though, you can check the style of other class names in this file; one should use CamelCase test names here if you were adding a new class.